src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -135,6 +135,15 @@
   <data name="XA3005" xml:space="preserve">
     <value>Could not link native shared library: {0}</value>
   </data>
+  <data name="XA4209" xml:space="preserve">
+    <value>Failed to generate Java type for class: {0} due to {1}</value>
+    <comment>{0} - The managed type name
+{1} - The exception message of the associated exception</comment>
+  </data>
+  <data name="XA4210" xml:space="preserve">
+    <value>You need to add a reference to Mono.Android.Export.dll when you use ExportAttribute or ExportFieldAttribute.</value>
```

Comment:
Should we also make these messages more polite while we're at it?  "You need to add..." sounds kinda rude; perhaps this should be "Please add..."?  Additionally, "you" should probably be removed.  Thus:

```xml
    <value>Please add a reference to Mono.Android.Export.dll when using ExportAttribute or ExportFieldAttribute.</value>
```

Reply:
Yeah, that sounds good. I'll add that.

I made a similar change to remove an exclamation point from another message to make it gentler. (That'll be coming up in another PR.)